Hyper/Window.cs

```diff
@@ -96,18 +110,17 @@ protected override void OnKeyDown(KeyboardKeyEventArgs e)
 
         if (e.Key == Keys.Escape)
```

Comment:
this behaves weird when starting the game, probably should return early if GeometryType==None or something